backend/src/data/migrations/20210304183602-create-users-table.js

```diff
@@ -49,9 +49,6 @@ module.exports = {
         allowNull: false,
         type: Sequelize.STRING
       },
-      diagnosis: {
-        type: Sequelize.UUID
-      },
       createdAt: Sequelize.DATE,
```

Comment:
You can't edit old migrations manually. If you need new changes - create the new one